sites/public/page_content/locale_overrides/general.json

```diff
@@ -21,5 +21,7 @@
   "pageDescription.resources": "The City of Detroit Housing and Revitalization Department has compiled a list of resources to help you find and maintain your housing. At the end of this page, you will also find four short videos where you can learn more about the application process for affordable housing.",
   "pageDescription.additionalResources": "We encourage you to browse additional resources in your search for housing.",
   "errors.minGreaterThanMaxRentError": "Min Rent must be less than or equal to Max Rent",
-  "errors.maxLessThanMinRentError": "Max Rent must be greater than or equal to Min Rent"
+  "errors.maxLessThanMinRentError": "Max Rent must be greater than or equal to Min Rent",
+  "t.additionalAccessibility": "Additional Accessibility Details",
```

Comment:
We can add this new key and its translations into the core translation files, not the overrides - so `en.json` and `es.json` etc in ui-components